docs/geedocs/5.3.0/answer/7160007.html

```diff
@@ -49,6 +49,14 @@ <h5>
       This can be done by configuring the <code>PURGE</code> and <code>PURGE_LEVEL</code> settings in <code>/etc/opt/google/XMLparams</code>. 
       This file must be created manually. View the <a href="../answer/176738.html">Configure Fusion performance</a> document for more details.</p>
     <p><strong>Added a <code>long_version</code> parameter to the SCons build. </strong>This will allow the user to override the long version string.</p>
+
+<p><strong>Added options to the Server install script.</strong>
+       The server install script now checks the size of the publish root and allows the user to set custom user and group names.
+</p>
+
+<p><strong>Improved system manager performance for large builds.</strong> This release eliminates some duplicate processing in system manager, which reduces the amount of time it takes to initiate large builds.
+</p>
+
```

Comment:
We probably should mention the upgrade of OpenJpeg 2000 library (beside the bug fix) part of new features or a section by itself for Updated libraries. We could mention that upgrade bring `bugs fix and improvements made into that new version`